assets/js/components/channels/ChannelNewCommunityIntro.jsx

```diff
@@ -0,0 +1,146 @@
+                background: "none",
+                fontWeight: 600,
+              }}
+              onClick={() => {
+                history.replace("/integrations/new");
+              }}
+            >
+              Back to New Integrations
+            </Button>
+            <div>
+              <Text style={{ fontSize: 27, fontWeight: 600 }}>
+                {integrationType.name}
+              </Text>
+            </div>
+          </div>
+          <div
+            style={{
+              padding: "25px 15px",
+              backgroundColor: "#ffffff",
+              height: "calc(100% - 100px)",
+              overflowY: "scroll",
+            }}
+          >
+            <img style={{ height: 60, width: 60 }} src={integrationType.img} />
+            <div style={{ marginTop: 5, marginBottom: 30 }}>
+              <Text><a href={integrationType.info.externalLink} target="_blank">Developer Website</a></Text>
+            </div>
+            {renderRow(integrationType, history, true)}
+          </div>
+        </MobileLayout>
+      </MobileDisplay>
+      <DesktopDisplay>
+        <DashboardLayout
+          title={integrationType.name}
+          user={props.user}
+          noAddButton
+          extra={<img style={{ height: 75, width: 75 }} src={integrationType.img} />}
+          breadCrumbs={
+            <div style={{ position: 'absolute', zIndex: 10 }}>
+              <Link
+                to="#"
+                onClick={e => {
+                  e.preventDefault()
+                  history.replace("/integrations/new")
+                }}
+              >
+                <Text strong><ArrowLeftOutlined style={{ fontSize: 12 }}/> Back to New Integrations</Text>
+              </Link>
+            </div>
+          }
+          underTitle={
+            <div style={{ position: 'absolute', zIndex: 10, bottom: 20 }}>
+              <Text><a href={integrationType.info.externalLink} target="_blank">Developer Website</a></Text>
+            </div>
+          }
+        >
+          <div
+            style={{
+              height: "100%",
+              width: "100%",
+              backgroundColor: "#ffffff",
+              borderRadius: 6,
+              overflow: "hidden",
+              boxShadow: "0px 20px 20px -7px rgba(17, 24, 31, 0.19)",
+            }}
+          >
+            <div style={{ overflowX: "scroll" }} className="no-scroll-bar">
+              <div style={{ padding: "60px 30px 60px 30px", minWidth }}>
+                {renderRow(integrationType, history)}
+              </div>
+            </div>
+          </div>
+        </DashboardLayout>
+      </DesktopDisplay>
+    </>
+  )
+}
+
+const renderRow = (integrationType, history, mobile = false) => (
+  <Row>
+    <Col span={mobile ? 24 : 12}>
+      <Text style={{ fontSize: 22, fontWeight: 600, display: 'block' }}>{integrationType.info.title}</Text>
+      <Text style={{ display: 'block', marginTop: 30 }}>{integrationType.info.desc}</Text>
```

Comment:
to match designs: 
title -> 28px
desc -> 16px

Reply:
made it smaller on mobile since 28 looks really ugly